src/ontology/mondo-edit.obo

```diff
@@ -428453,12 +428453,20 @@ created_by: http://orcid.org/0000-0001-5208-3432
 id: MONDO:0100096
 name: 2019 novel coronavirus infection
 def: "Coronavirus disease 2019 (COVID-19) is a respiratory illness that can spread from person to person. The virus that causes COVID-19 is a novel coronavirus that was first identified during an investigation into an outbreak in Wuhan, China." [https://www.cdc.gov/coronavirus/2019-ncov/downloads/2019-ncov-factsheet.pdf]
+synonym: "2019-nCoV (novel coronavirus) detected" RELATED [SCTID:1240581000000104]
 synonym: "2019-nCoV infection" EXACT [https://www.cdc.gov/coronavirus/2019-ncov/index.html]
 synonym: "COVID-19" EXACT [https://www.cdc.gov/coronavirus/2019-ncov/about/index.html]
+synonym: "disease caused by 2019-nCoV (novel coronavirus)" EXACT [] {comment="SCTID:1240751000000100"}
 synonym: "SARS-CoV-2" EXACT [doi:10.1038/s41564-020-0695-z]
```

Comment:
our DPs say "% infection"

Reply:
We do have % infection as synonyms.

I'm unclear what the action item here is - do we not want this synonym?

Reply:
From our conversation on Skype - my understanding is we do not want to add these synonyms, we just want the xrefs.